layers/meta-resin-raspberrypi/recipes-bsp/u-blox-modeswitch/files/u-blox-switch.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+while true; do
```

Comment:
Why do you need an endless loop? I'm confused on how you break it - other than the reboot command.

Reply:
The U-Blox modem is kind of stubborn and sometimes after it restarts, it is still in RNDIS mode instead of ECM mode.
So we need to tell him again to restart in ECM mode, and we check with the lsusb command if it restarted in ECM mode.
If the modem doesn't go into ECM mode, the board will not be able to connect to openvpn. The client will have a board in a remote place, that it can't access. 
This is why we are retrying this mode switch in an infinite loop.
Sometimes mode switch works from the first time but it happened to take 16 restarts.

Reply:
I understood now how it works.